search-api/src/main/scala/no/ndla/searchapi/service/search/MultiDraftSearchService.scala

```diff
@@ -172,6 +172,7 @@ trait MultiDraftSearchService {
             simpleStringQuery(queryString.underlying).field("grepContexts.title", 1),
             nestedQuery("contexts", boolQuery().should(termQuery("contexts.contextId", queryString.underlying)))
               .ignoreUnmapped(true),
+            boolQuery().should(termQuery("contextids", queryString.underlying)),
```

Comment:
Trenger dette å være en `boolQuery`?
Kan ikke denne bare ligge direkte i den over?

```suggestion
            termQuery("contextids", queryString.underlying),
```

Reply:
Testene blei grønne ved å droppe boolQuery så gjør det slik.